website/docs/r/s3_bucket_policy.html.markdown

```diff
@@ -15,36 +15,33 @@ Attaches a policy to an S3 bucket resource.
 ### Basic Usage
 
 ```terraform
-resource "aws_s3_bucket" "b" {
+locals {
+  aws_account_b_id = "123456789012"
+}
```

Comment:
Though I understand why you'd do so, I don't think adding a `locals` block here is necessary, and may be confusing for less experienced Terraform practitioners; particular since it's only being used once in the example. We can omit this and pass it directly below.

```suggestion
```

Reply:
yep, this makes total sense - the logic behind adding the local was to ensure that the reader knew that the string of numbers was an account ID and not some other piece of AWS deep lore, but i figure that we have to assume some level of familiarity (or ability to look things up) with regards to AWS.

Will update :)